pennylane/transforms/core/transform.py

```diff
@@ -272,13 +273,31 @@ def circuit():
         representation of the function being transformed stored in the ``inner_jaxpr``
         parameter of the transform's primitive.
 
+        **Fallback implemementation of plxpr transforms:**
+
+        If a transform that does not define a ``plxpr_transform`` is applied to a function,
+        a fallback implementation of the transform is used. This fallback implementation converts
+        the function into a :func:`~pennylane.tape.QuantumScript`, which is then transformed
+        as usual. However, because of the constraints of program capture, many transforms will not
+        be compatible with this fallback implementation:
+
+        * Transforms that return multiple tapes are not compatible.
+        * Transforms that require non-trivial postprocessing of results are not compatible.
+        * Dynamically shaped arrays are not compatible.
+        * Functions that are being transformed that contain control flow dependent on dynamic
+          parameters are not compatible. This includes:
+
+          * :func:`pennylane.cond` with dynamic parameters as predicates.
+          * :func:`pennylane.for_loop` with dynamic parameters for ``start``, ``stop``, or ``step``.
+          * :func:`pennylane.while_loop` does not work.
+
         .. warning::
 
             Currently, executing a function to which a transform has been applied will raise a
             ``NotImplementedError``. See below for details on how to use functions that are
             transformed.
 
-        To apply the transform, the :func:`pennylane.capture.expand_plxpr_transforms` function
+        To perform the transform, the :func:`pennylane.capture.expand_plxpr_transforms` function
```

Comment:
@mudit2812 is this needed for tape transforms (no plxpr implementation) that are being applied when capture is enabled?

Reply:
Yep, because we're going to treat all transforms as plxpr transforms.

Reply:
So, if a user wants to (a) use a "core" tape transform that's not supported with plxpr yet, or (b) use a custom tape transform they've written with capture enabled, they **_have to_** decorate with `expand_plxpr_transforms`?

Reply:
Yep. Any transform that doesn't have a `plxpr_transform` will still get registered as having a "fallback" `plxpr_transform`, and the "fallback" plxpr transform will take jaxpr as input, and return transformed jaxpr. The fact that we convert the jaxpr to a tape, apply the transform, and then convert the transformed tape back into jaxpr, is an internal detail.

Reply:
Is there no way to hide `expand_plxpr_transforms` here?